app/models/video.rb

```diff
@@ -4,6 +4,7 @@ class Video < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
   include ::OregonDigital::TriplePoweredProperties::WorkBehavior
   include ::OregonDigital::VideoMetadata
+  include ::OregonDigital::GenericMetadata
```

Comment:
included at the bottom of the file, which one should remain..remove one or the other